twiter/logic.js

```diff
@@ -0,0 +1,70 @@
+const Tweeter = function () {
+
+    const posts = [{ id: 1, txt: "I am the First", comments: [], commentIdCounter: 0 }]
+    let postIdCounter = 1
+
+    const getPosts = function () {
+        return posts
+    }
+
+
+    const getComments = function (postID,) {
+        let go = getPosts()
+        for (let i = 0; i < go.length; i++) {
+            if (go[i].id == postID) {
+                let com = go[i].comments
+                return com
+
+            }
+        }
+    }
+
+
+    const addPost = function (text) {
+        postIdCounter++
+        let somePost = { id: postIdCounter, txt: text, comments: [], commentIdCounter: 0 }
```

Comment:
use `const` as much as you can. 

`const somePost = ` you can use const because you are just editing the information inside the object. Not changing the fact that `somePost` is an object